### DISCUSSION TITLE
Assuming CVE-2024-7971 affect CefSharp - Is there expected update for Chromium >128.0.6613.84?

### DISCUSSION THREAD

**Discussion by idohcl** [cefsharp/CefSharp#4928] [Q&A] [answered]
There is a known high severity security vulnerability: CVE-2024-7971 in Chromium.
Assuming it affects CefSharp - Is there expected update for Chromium >128.0.6613.84?

Reference: https://nvd.nist.gov/vuln/detail/CVE-2024-7971

**Comment by TamirHCL** [comment]


**Comment by nir-moshe** [comment]
please release 128 ASAP 🙂

**Comment by amaitland** [comment, marked as answer]
A new release will be out shortly. Subscribe to release only notifications here on github to be notified when it's out.

**Reply by idohcl** [reply]
Thank you @amaitland 
It is highly appreciated
